backend/src/util.rs

```diff
@@ -5,13 +5,20 @@
 use std::fmt::Display;
 
 use async_graphql::Enum;
+use image::DynamicImage;
 
 /// Date type used in multiple places.
 pub type Date = chrono::NaiveDate;
 
-// Uuid type used in multiple places.
+/// Uuid type used in multiple places.
 pub type Uuid = uuid::Uuid;
 
+/// An in memory representation of an image containing pixels.
+pub type ImageResource = DynamicImage;
+
+/// File extension used for images
+pub const IMAGE_EXTENSION: &str = "jpg";
```

Comment:
If we want to change that extension in production, is it possible that "older" images (*.jpg) can be processed with the new type? And if not, is there a generalized image type, that would prevent such issue?

Reply:
the webserver will serve images as they are in the directory but you would need to specify the correct extension on request. when saving the image it will be saved with the specified format/extension. Our image Type is independent of encoding (how it is stored on disk).

Reply:
Ok. Then I assume we never change the extension, as we then can't request all the existing images.

Reply:
We could add a database attribute for that but we can also do then when we really want to change filetypes in the future.